algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,47 @@
+const bfs = (graph, root, targetNode) => {
+
+  buildPath = (parents, targetNode) => {
```

Comment:
Codacy found an issue: ['buildPath' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6319059)